docs/internal-documentation/maintenance/ReleaseProcess.md

```diff
@@ -92,6 +109,8 @@ MANIFEST.SKIP, i.e. no missing or extra files:
 
 ## 7. Produce distribution tarballs
 
+This section is not relevant for Zonemaster-GUI.
```

Comment:
Yes, we have a distribution tarballs. 

The requirements are nodejs and npm. There are available from the [official website]( https://nodejs.org/en/).
Minimal version of Nodejs is 6.0 but install the last LTS version available.

To build the tarballs, steps are: 

1. `git clone https://github.com/zonemaster/zonemaster-gui.git`  
2. `cd zonemaster-gui`
3. `npm install` 
4. `npm run release`

The distribution tarballs is in the root level of the zonemaster-gui folder. Its name is `zonemaster_web_gui.zip`.

Reply:
Fixed.